### ISSUE TITLE
Nested bars: Unexpected keyword argument 'position'

### ISSUE THREAD

**Issue by neal-morgansolar** [tqdm/tqdm#130]
I'm getting the following error when trying to use nested bars:

```
    for filename in tqdm(filenames, desc="Files", position=0):
TypeError: __init__() got an unexpected keyword argument 'position'
```

If I leave the `position` argument out, the only one bar shows. The code I'm trying to run is below:
```python
for filename in tqdm(filenames, desc="Files", position=0):
    wb = openpyxl.load_workbook(os.path.join(data_dir, filename))
    sheet = wb.worksheets[0]
    for _, things in blocks.items():
        for row in tqdm(sheet.iter_rows(things), desc="cells", position=1):
            for cell in row:
                # do things
```

Using the following example code from the documentation:

```python
from tqdm import trange
from time import sleep

for i in trange(10, desc='1st loop'):
    for j in trange(5, desc='2nd loop', leave=False):
        for k in trange(100, desc='3nd loop'):
            sleep(0.01)
```

Produces this bar only this one bar:
![tqdm-cmder](https://cloud.githubusercontent.com/assets/14981598/13437060/57d9a822-dfb0-11e5-8e55-9339f4797699.png)

Colorama is installed in the same venv. 

Using python3.4 on windows 7.

**Comment by neal-morgansolar** [comment]
Tested on an Ubuntu VM as well, exact same result.

**Comment by casperdcl** [comment]
Ah can you try v4.0.0 from github? I think this was implemented in 4.0.0 but broken in v4.1.0 by e0001362b17970e7c5c2f87f5abbc48d333eb5d8

**Comment by neal-morgansolar** [comment]
I tried running the same example code with v4.0.0, this is the result:

![2016-03-02 13_50_12-cmder](https://cloud.githubusercontent.com/assets/14981598/13471238/c0aad376-e07d-11e5-9403-4e5a02bb9cda.png)

It continually scrolls up. All three bars do show, but the first and second bars are pretty much invisible while is scrolls quickly.

**Comment by casperdcl** [comment]
Hmm the screenshot is exactly what you'd expect if the up character was ignored in your environment but the return and line feed (\r and \n) are. What environment are you using? If I can reproduce your issue I might be able to help

**Comment by neal-morgansolar** [comment]
I'm using windows 7, python 3.4. 

Honestly, if it's not a huge concern for you then don't worry about it. I just figured I would report it to you. I can make do just fine with a single progress bar (which works great).

**Comment by Jessime** [comment]
Maybe this should be it's own issue, but I'm having very similar problems. I'm running the Jupyter Notebook 4.1.0 on Ubuntu 14.04 LTS with tqdm  4.3.0 (installed today). I see:

![tqdm](https://cloud.githubusercontent.com/assets/8493770/13683692/0d12771a-e6d6-11e5-8a79-303accdb9376.png)

**Comment by lrq3000** [comment]
@Jessime @cesans Please see #134. This issue in Jupyter is because of Jupyter limited console wrapper, but we will soon provide a native Jupyter widget to workaround this issue.

**Comment by casperdcl** [comment]
Closed in favour of #134
